stac_fastapi/pgstac/core.py

```diff
@@ -226,6 +226,9 @@ async def _get_base_item(collection_id: str) -> Dict[str, Any]:
 
             for feature in collection.get("features") or []:
                 base_item = await base_item_cache.get(feature.get("collection"))
+                # Exclude None values
+                base_item = {k: v for k, v in base_item.items() if v is not None}
```

Comment:
from https://github.com/stac-utils/pgstac/pull/263, hydration does not handle `None` value, so a quick fix is to manually remove it

Reply:
cc @kylebarron @bitner